packages/blockchain-wallet-v4-frontend/src/components/BuySell/Coinify/Payment/index.js

```diff
@@ -57,9 +48,8 @@ class PaymentContainer extends Component {
           handlePaymentClick={this.handlePaymentClick}
           medium={this.state.medium}
           quote={this.props.quote}
-          triggerKyc={this.triggerKyc}
+          triggerKyc={() => showModal(KYC_MODAL)}
           busy={busy}
-          openPendingKyc={openKYC}
           handlePrefillCardMax={limits => checkoutCardMax(limits)}
```

Comment:
can use(this action ignores any arguments but the first)
```
handlePrefillCardMax={checkoutCardMax}
```

Reply:
for perf reasons it's always best to not create any extra functions in `cata`

Reply:
Didn't know that - cool!